### PR TITLE
Cleaned up some boilerplate and improved cosmetic UI

### PR DESCRIPTION
**Pull request by Nibbl-z**
- added Utilities.getLocationFromConfig, which gets a location... from the mapdata... because in sniffer caretaker and other minigames the calls to get the locations would be like 20 lines each
- made cosmetic inventory sort by rarity
- added click sfx to cosmetic ui

### CODE REVIEW

**Comment by Nibbl-z** [comment]
Sorry for not making new cosmetics i had to like do my homework i didnt have enough time 💔

**Comment by Nibbl-z** [comment]
actually... hmm,, im gonna stay up past my bedtime and make ONE because im EVIL

**Comment by Nibbl-z** [comment]
nvm ill do it tomorrowwwwwwwwwwwwwwwwwwwwwwwwww

File: src/main/java/xyz/devcmb/cmr/minigames/KaboomersController.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package xyz.devcmb.cmr.minigames;
 
+import jdk.jshell.execution.Util;
```
Comment (29cmb) [inline src/main/java/xyz/devcmb/cmr/minigames/KaboomersController.java:3]: unused import statement my beloved

File: src/main/java/xyz/devcmb/cmr/minigames/SnifferCaretakerController.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package xyz.devcmb.cmr.minigames;
 
+import jdk.jshell.execution.Util;
```
Comment (29cmb) [inline src/main/java/xyz/devcmb/cmr/minigames/SnifferCaretakerController.java:3]: again

**Review by 29cmb — DISMISSED** [review]
lgtm

**Comment by Shuflduf** [comment]
g

**Comment by 29cmb** [comment]
> g

Approved

**Pull request merged**